tuna/miopen/miopen_lib.py

```diff
@@ -170,7 +144,7 @@ def parse_args(self):
                        default=None,
                        help='execute on each machine')
 
-    self.clean_args()
+    clean_args()
```

Comment:
If `args` is owned by `MIOpen` then `clean_args` should be part of it as well ?

Reply:
its a generic function because its also called by example and pylint complains of duplicate code